doc/source/ray-air/api/configs.rst

```diff
@@ -1,25 +0,0 @@
-
-Ray AIR Configurations
-======================
```

Comment:
assuming this already got moved to train?

Reply:
Yep, the configs are in the Train/Tune API references already.